lib/logtail-rack/http_events.rb

```diff
@@ -142,7 +142,7 @@ def call(env)
 
             Config.instance.logger.info do
               http_context = CurrentContext.fetch(:http)
-              content_length = headers[CONTENT_LENGTH_KEY]
+              content_length = (headers[CONTENT_LENGTH_KEY].to_i unless headers[CONTENT_LENGTH_KEY].nil?)
```

Comment:
This will define `content_length` even if `headers[CONTENT_LENGTH_KEY].nil?` right?

Reply:
We could also extract this to a func as it's used three times, what do you think @simonrozsival ?